tests/components/hue/conftest.py

```diff
@@ -0,0 +1,11 @@
+"""Test helpers for Hue."""
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def no_request_delay():
+    """Make the request refresh delay 0 for instant tests."""
+    with patch("homeassistant.components.hue.light.REQUEST_REFRESH_DELAY", 0):
```

Comment:
We may need to patch the sensor and binary_sensor coordinator delay too.

Reply:
Sensors never request an update because they cannot be controlled. Didn't seem to impact the tests.

Reply:
Yes.